src/Umbraco.Forms.Integrations.Commerce.EMerchantPay/App_Plugins/UmbracoForms.Integrations/Commerce/eMerchantPay/customer-details-mapper.controller.js

```diff
@@ -32,12 +32,16 @@
 
     function init() {
 
-        umbracoFormsIntegrationsCommerceEMerchantPayResource.getMappingFields().then(function (response) {
+        umbracoFormsIntegrationsCommerceEMerchantPayResource.getDefaultMappingFields().then(function (response) {
```

Comment:
Maybe rename to `getAvailableMappingFields`?  As perhaps that expresses what they are a bit better.

Reply:
Updated